addons/sys_core/XEH_postInit.sqf

```diff
@@ -151,4 +151,9 @@ if (getClientStateNumber < 10) then { // Check before game has started (in brief
     }, 0, []] call CBA_fnc_addPerFrameHandler;
 };
 
+["CBA_loadoutGet", {
```

Comment:
Can you move this before the `!hasInterface` so when `CBA_fnc_getLoadout` is called server side, it catch the correct loadout?

Reply:
@BrettMayson ?